chore(vision-page-plugin): remove build folder, change author to UT OCM

corbsch opened the pull request:
Remove `build` folder
Change author in `package.json` to **UT OCM**

@naylormar Can you think of anything else we could add to this since it's more of a generic PR? I thought about moving in the formatting stuff we had on some of the other plugins, but that could wait since we don't really have a final process of how that should work.

corbsch committed:
chore(vision-page-plugin): remove build folder, change author to UT OCM

naylormar reviewed (approved):
Wow! Not my best repo setup work 😂. Good catch on the build folder.

corbsch committed:
chore(vision-page-plugin): change author

corbsch commented:
> Wow! Not my best repo setup work 😂. Good catch on the build folder.

@naylormar It's all good! Super easy fix. I think it only happened because you had forgotten the `.gitignore` on the initial setup.

FYI - I added another commit where I changed the name to the recommended one (via @pikemurdy) after you approved.
